js-miniapp-bridge/src/common-bridge.ts

```diff
@@ -647,6 +648,63 @@ export class MiniAppBridge {
       );
     });
   }
+
+  /**
+   * In-App Purchases Implementation
+   */
+  /**
+   * This will retrieve the list of products deatils available for inapp-purchases from the play/app stores.
+   * @returns List of products for inapp-purchases
+   * @see {getProducts}
+   */
+  getProducts() {
+    return new Promise<Product[]>((resolve, reject) => {
+      return this.executor.exec(
+        'requestGetProducts',
```

Comment:
Are we going to add "request" prefix to all those In-App purchase related functions? It will be reflected in native side as well.

Reply:
I have just followed the existing pattern, also as the MiniApps in performs a request for something to host app I think it make sense t prefix the "request" word.
Let me know if this can be changed to something better.

Reply:
@GondiTilak 
cc @rleojoseph @Climbatize 
the documentation probably needs an update unfortunately I think we don't need the `getProducts` interface as it should be `prepareProducts` (https://confluence.rakuten-it.com/confluence/display/ONEAPP/Documentation+-+IAP) which tells if the products were validated on host side. the miniapp should know which product identifiers to send. we can discuss later.

Reply:
@GondiTilak yeah, timo is right, we dont need product list on native side, to purchase a product from native side we just need the product id whichever product is selected by the user.so i think the flow should be
`User will select a product on miniapp-> miniapp will invoke sdk.purchaseItem(id) -> invoke native side -> native side will purchase using billing -> callback to miniapp.`